### DISCUSSION TITLE
Fdroid Mirror Select

### DISCUSSION THREAD

**Discussion by n4b7l** [NeoApplications/Neo-Store#536] [General]
Is it possible to use f-droid repo mirror? Can I change the default repo URL? How Neo Store decides which f-droid mirror to use?

**Comment by machiav3lli** [comment]
(Your question has been already answered in the community groups, afaik.)
It's possible to manually change the repo's mirror. But some people seem to face a bug where the bottom of the repo page isn't visible so they can't change or edit. Rotating the screen has fixed this for users who reported this issue.
